### ISSUE TITLE
TextFormField Attempted to get clipboard data that requires additional permission(s).

### ISSUE THREAD

**Issue by jeancarlos2020** [flutter/flutter#72217]
```dart
class RegistroPage extends StatefulWidget {
  @override
  _RegistroPageState createState() => _RegistroPageState();
}

class _RegistroPageState extends State<RegistroPage> {
  @override
  Widget build(BuildContext context) {
    GlobalKey<FormState> formKey = GlobalKey();
    return Scaffold(
      body: Container(
        child: Form(
          key: formKey,
          child: TextFormField(),
        ),
      ),
    );
  }
}
```

**Actual results:** 

```
W/PlatformPlugin(18730): Attempted to get clipboard data that requires additional permission(s).
W/PlatformPlugin(18730): See the exception details for which permission(s) are required, and consider adding them to your Android Manifest as described in:
W/PlatformPlugin(18730): https://developer.android.com/guide/topics/permissions/overview
W/PlatformPlugin(18730): java.lang.SecurityException: Permission Denial: opening provider org.chromium.chrome.browser.util.ChromeFileProvider from ProcessRecord{e20c83d 18730:com.cgz.usuarios/u0a420} (pid=18730, uid=10420) that is not exported from UID 10138
W/PlatformPlugin(18730): 	at android.os.Parcel.createException(Parcel.java:1953)
W/PlatformPlugin(18730): 	at android.os.Parcel.readException(Parcel.java:1921)
W/PlatformPlugin(18730): 	at android.os.Parcel.readException(Parcel.java:1871)
W/PlatformPlugin(18730): 	at android.app.IActivityManager$Stub$Proxy.getContentProvider(IActivityManager.java:4067)
W/PlatformPlugin(18730): 	at android.app.ActivityThread.acquireProvider(ActivityThread.java:6086)
W/PlatformPlugin(18730): 	at android.app.ContextImpl$ApplicationContentResolver.acquireUnstableProvider(ContextImpl.java:2617)
W/PlatformPlugin(18730): 	at android.content.ContentResolver.acquireUnstableProvider(ContentResolver.java:1841)
W/PlatformPlugin(18730): 	at android.content.ContentResolver.openTypedAssetFileDescriptor(ContentResolver.java:1455)
W/PlatformPlugin(18730): 	at android.content.ContentResolver.openTypedAssetFileDescriptor(ContentResolver.java:1415)
W/PlatformPlugin(18730): 	at io.flutter.plugin.platform.PlatformPlugin.getClipboardData(PlatformPlugin.java:297)
W/PlatformPlugin(18730): 	at io.flutter.plugin.platform.PlatformPlugin.access$700(PlatformPlugin.java:27)
W/PlatformPlugin(18730): 	at io.flutter.plugin.platform.PlatformPlugin$1.getClipboardData(PlatformPlugin.java:86)
W/PlatformPlugin(18730): 	at io.flutter.embedding.engine.systemchannels.PlatformChannel$1.onMethodCall(PlatformChannel.java:141)
W/PlatformPlugin(18730): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:233)
W/PlatformPlugin(18730): 	at io.flutter.embedding.engine.dart.DartMessenger.handleMessageFromDart(DartMessenger.java:85)
W/PlatformPlugin(18730): 	at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(FlutterJNI.java:735)
W/PlatformPlugin(18730): 	at android.os.MessageQueue.nativePollOnce(Native Method)
W/PlatformPlugin(18730): 	at android.os.MessageQueue.next(MessageQueue.java:326)
W/PlatformPlugin(18730): 	at android.os.Looper.loop(Looper.java:165)
W/PlatformPlugin(18730): 	at android.app.ActivityThread.main(ActivityThread.java:6831)
W/PlatformPlugin(18730): 	at java.lang.reflect.Method.invoke(Native Method)
W/PlatformPlugin(18730): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:547)
W/PlatformPlugin(18730): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:927)
W/PlatformPlugin(18730): Caused by: android.os.RemoteException: Remote stack trace:
W/PlatformPlugin(18730): 	at com.android.server.am.ActivityManagerService.getContentProviderImpl(ActivityManagerService.java:12867)
W/PlatformPlugin(18730): 	at com.android.server.am.ActivityManagerService.getContentProvider(ActivityManagerService.java:13179)
W/PlatformPlugin(18730): 	at android.app.IActivityManager$Stub.onTransact(IActivityManager.java:358)
W/PlatformPlugin(18730): 	at com.android.server.am.ActivityManagerService.onTransact(ActivityManagerService.java:3408)
W/PlatformPlugin(18730): 	at android.os.Binder.execTransact(Binder.java:726)

```
**Flutter doctor:** 

Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel master, 1.24.0-8.0.pre.321, on Mac OS X 10.15.7 19H15 darwin-x64, locale es-419)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
[✓] Xcode - develop for iOS and macOS (Xcode 12.2)
[✓] Android Studio (version 4.1)
[✓] VS Code (version 1.51.1)
[✓] Connected device (1 available)

**Comment by hamdikahloun** [comment]
Hi @jeancarlos2020 
This is the right behavior .
For more information see : https://github.com/flutter/flutter/issues/66108
Can you please try this https://github.com/flutter/flutter/issues/66108#issuecomment-695218268 ?
Thank you

**Comment by pedromassangocode** [comment]
Hi @jeancarlos2020 
Please try what were asked above.

**Comment by jeancarlos2020** [comment]
My solution is command flutter clean, THANKS.

**Comment by duzenko** [comment]
TL;DR;
Reset your device clipboard (by copying any plain text into it) or add external storage permission to the Android Manifest of you like the hard ways
